pkg/apis/serving/v1alpha1/revision_types.go

```diff
@@ -129,13 +113,6 @@ type RevisionSpec struct {
 	// +optional
 	DeprecatedServingState DeprecatedRevisionServingStateType `json:"servingState,omitempty"`
 
-	// DeprecatedConcurrencyModel specifies the desired concurrency model
-	// (Single or Multi) for the
-	// Revision. Defaults to Multi.
-	// Deprecated in favor of ContainerConcurrency.
-	// +optional
-	DeprecatedConcurrencyModel RevisionRequestConcurrencyModelType `json:"concurrencyModel,omitempty"`
```

Comment:
I'm ok with dropping fields from our internal APIs, which we've reconciled out of any existing internal resources, but we should leave the fields in our public API until it is turned down.